openfl-tutorials/experimental/workflow_interface_requirements.txt

```diff
@@ -2,8 +2,15 @@ astor==0.8.1
 chardet
 charset-normalizer
 dill==0.3.6
+matplotlib>=2.0.0
 metaflow==2.7.15
+mistune>=2.0.3 # not directly required, pinned by Snyk to avoid a vulnerability
 nbdev==2.3.12
+numpy>=1.13.3
+openfl>=1.2.1
```

Comment:
Any reason for having the minimum required version for openfl to be this old?

Reply:
resolved in e1e711efeb4f84ed325d0ae3401328c80502479a